libraries/AP_Scripting/AP_Scripting_CANSensor.h

```diff
@@ -38,6 +40,10 @@ class ScriptingCANSensor : public CANSensor {
     ScriptingCANSensor(AP_CAN::Protocol dtype)
         : CANSensor("Script") {
         register_driver(dtype);
+        for (int i=0;i<MAX_FILTERS;i++) {
+            filter_mask[i] = UINT32_MAX;
+            filter_value[i] = UINT32_MAX;
```

Comment:
Surely the default value would be zeros? Anything & 0 == 0, so it would pass  everything?